Resources/public/js/recommendationtemplatehelper.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+/**
+ * Converts ISO-8601 string to locale time/date format.
```

Comment:
![ISO 8601 was published on 06/05/88 and most recently amended on 12/01/04.](https://imgs.xkcd.com/comics/iso_8601.png)